Sources/Sentry/SentryHook.c

```diff
@@ -0,0 +1,111 @@
+#include "SentryHook.h"
```

Comment:
`m`: I think we should put these files and the fishhook files into `Sentry/SentryCrash/Recording/Tools`